pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
@bgrant0607 I can't see a circumstance where it'll ever differ for a cloud platform, and I can't see why one would care if you didn't have createExternalLB.
Should we just teach the proxy about cloud providers and let it get the value directly from the code?